structure/0006-governance.md

```diff
@@ -2,53 +2,135 @@
 
 # Explanation
 
-## Governmental Structure
-![Governmental Structure](../figures/quilt-goverance-diagram.png)
-\* For illustration purposes only, technical teams may vary
+## Teams
 
-### Teams
-A team is a group of staff members dedicated to solving a specific issue, maintaining a specific subproject, or overseeing a specific area. A team may be made up of any number of staff members, and a staff member may be on any number of teams. Teams may be community-focused, such as community moderators, or technically focused, such as the team of developers responsible for Quilt Loader. Team member responsibilities will vary and be detailed on a team-by-team basis.
+A team is a group of team members dedicated to solving a specific issue,
+maintaining a specific sub project or overseeing a specific area.
 
-Each team will be initially constructed by the parent team responsible for that team. Further appointments will be voted upon by team members.
+All teams exist as teams on GitHub and can be viewed at https://github.com/orgs/QuiltMC/teams.
 
-#### Voting
-When a vote is called within a team, each member of that team may choose to vote for any of the available options or abstain. The option with the plurality of votes wins. Members of supervising teams may choose to participate in votes. In the event of a tie, members of the parent team may choose to vote if they haven't already, with each of their votes being worth three votes.
+### Team Members
 
-### Removal of Staff
-Any team member (up to and including administrative board members) may be removed by a majority vote of all staff members above and below.
- * For members of the administrative board, this would be all staff members
- * For technical leads, this would be all technical staff members, as well as all members of the administrative board
- * For the QSL Module A lead, this would be all team members of QSL Module A, all technical leads, and all members of the administrative board
- * For a community moderator, this would be all community moderators and all members of the administrative board
+A team member is a member of the organization assigned to a specific team on GitHub.
+A member of the organization may be assigned to multiple teams.
+A team must have at least one member.
+
+### Team Leads
+
+A team lead is a team member with the Team Maintainer role for the relevant team on GitHub.
+A member of the organization may be a team lead for multiple teams.
+Each team must have at least one team lead.
+
+### Team Responsibilities
+
+Team responsibilities will vary and be detailed on a team-by-team basis.
+In general, a team is responsible for performing the tasks outlined in the RFC that proposed its creation.
+
+Most teams will also maintain one or more repositories on GitHub.
+The team is responsible for reviewing pull requests and ensuring that they meet the Quilt projects goals in quality and purpose.
```

Comment:
The correct way to refer to the project is capitalised - so, `The Quilt Project`. Also, you missed an apostrophe - `The Quilt Project's`